README.md

```diff
@@ -2,13 +2,24 @@
 
 Toitū Te Whenua makes New Zealand's most up-to-date publicly owned aerial imagery freely available to use under an open licence. You can access this through the [LINZ Data Service](https://data.linz.govt.nz/data/category/aerial-photos/?s=n), [LINZ Basemaps](https://basemaps.linz.govt.nz/#@-41.8899962,174.0492437,z5) or the [Registry of Open Data on AWS](https://registry.opendata.aws/nz-imagery/).
 
+## Quickstart
+
+[<img src="./docs/img/stac-logo.png" width="120" />](https://stacindex.org/catalogs/nz-imagery#/)
+
+Browse the archive with [STAC Index](https://stacindex.org/catalogs/nz-imagery#/) or access the catalog directly 
+
+```https://nz-imagery.s3-ap-southeast-2.amazonaws.com/catalog.json```
```

Comment:
Would it be worth it making this a clickable link?

Reply:
I was going back and forward on this, and switched back to a clickable link.